app/extensions/brave/locales/bn-BD/preferences.properties

```diff
@@ -384,3 +384,4 @@ tabPreviewTiming=Time to wait before previewing a tab
 long=Long
 normal=Normal
 short=Short
+useIPFS=Enable IPFS (requires browser restart)
```

Comment:
I believe this is no longer necessary as the preference pane doesn't show this message anymore.